app-shells/fish/fish-2.5.0.ebuild

```diff
@@ -20,8 +20,12 @@ RDEPEND="
 DEPEND="
 	${RDEPEND}
 	sys-devel/bc
-	sys-devel/gettext
+	nls? ( sys-devel/gettext )
 "
+src_prepare() {
+	eapply "${FILESDIR}/${P}-honor-linguas.patch"
+	default
+}
```

Comment:
btw, I believe it would be great to have it documented here
https://devmanual.gentoo.org/ebuild-writing/variables/index.html
It's the first place newbies like me check for valid variables.

I't indirectly mentioned here in PMS
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-1010009.1.5
and also mentioned in this and other eclass pages.
https://devmanual.gentoo.org/eclass-reference/base.eclass/index.html
but it takes some effort to reach this.

I'll try to suggest it by opening a devmanual bug.